axoproject/src/generic.rs

```diff
@@ -71,9 +71,9 @@ impl std::fmt::Display for WorkspaceMember {
         match self {
             WorkspaceMember::Generic(path) => write!(f, "{MEMBER_GENERIC}:{path}"),
             #[cfg(feature = "cargo-projects")]
-            WorkspaceMember::Cargo(path) => write!(f, "{MEMBER_CARGO}/{path}"),
+            WorkspaceMember::Cargo(path) => write!(f, "{MEMBER_CARGO}:{path}"),
             #[cfg(feature = "npm-projects")]
-            WorkspaceMember::Npm(path) => write!(f, "${MEMBER_NPM}/{path}"),
+            WorkspaceMember::Npm(path) => write!(f, "${MEMBER_NPM}:{path}"),
```

Comment:
This change is to address what appears to be a bug in axoproject.

Basically, without this PR, `WorkspaceMember::from_string()` wanted `{KIND}:{PATH}`, but `WorkspaceMember::to_string()` generated `{KIND}/{PATH}` for everything but generic projects.

This meant, in practice, regenerating TOML files for any Cargo or NPM projects was _guaranteed_ to fail.

Basically, this could be successfully loaded from TOML:

```
[workspace]
members = ["cargo:*"]
```

But converting it back to TOML representation would yield this incorrect output:

```
[workspace]
members = ["cargo/*"]
```

Reply:
oh wild, i do not remember this decision being a thing so i can't add context, but picking one seems very good to me and i think : is better than /